packages/vaadin-themable-mixin/vaadin-theme-property-mixin.d.ts

```diff
@@ -10,27 +10,6 @@ export declare function ThemePropertyMixin<T extends Constructor<HTMLElement>>(
 ): Constructor<ThemePropertyMixinClass> & T;
 
 export declare class ThemePropertyMixinClass {
-  /**
-   * Helper property with theme attribute value facilitating propagation
-   * in shadow DOM.
-   *
-   * Enables the component implementation to propagate the `theme`
-   * attribute value to the sub-components in Shadow DOM by binding
-   * the sub-component’s "theme" attribute to the `theme` property of
-   * the host.
-   *
-   * **NOTE:** Extending the mixin only provides the property for binding,
-   * and does not make the propagation alone.
-   *
-   * See [Styling Components: Sub-components](https://vaadin.com/docs/latest/styling/custom-theme/styling-components/#sub-components).
-   * page for more information.
-   *
-   * @deprecated The `theme` property is not supposed for public use and will be dropped in Vaadin 24.
-   * Please, use the `theme` attribute instead.
-   * @protected
-   */
-  theme: string | null | undefined;
```

Comment:
nit: The `_theme` property below actually ends up in the public API of the component. Should be declared as `protected readonly _theme...`
<img width="855" alt="Screenshot 2022-11-15 at 11 51 05" src="https://user-images.githubusercontent.com/1222264/201888262-dad76fcc-4a95-4131-9ed5-2873750a87ac.png">

Reply:
Good point, thanks. Added `protected`.